cmd/root/sweet.go

```diff
@@ -41,20 +41,24 @@ type viewOptions struct {
 }
 
 type styles struct {
-	commentStyle lg.Style
-	untypedStyle lg.Style
-	cursorStyle  lg.Style
-	typedStyle   lg.Style
-	mistakeStyle lg.Style
+	commentStyle         lg.Style
+	untypedStyle         lg.Style
+	cursorStyle          lg.Style
+	typedStyle           lg.Style
+	mistakeStyle         lg.Style
+	vignetteStyle        lg.Style
+	vignetteMistakeStyle lg.Style
```

Comment:
This property is no longer relevant, since I'm no longer vignetting the first line of the exercise